apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -24,6 +24,15 @@ export default class ColorPickerPropertyRow extends React.Component {
     window.removeEventListener('mousedown', this.handlePageClick);
   }
 
+  componentWillReceiveProps(nextProps) {
+    const {initialValue} = nextProps;
+    if (this.props.initialValue !== initialValue) {
+      this.setState({
+        value: initialValue
+      });
+    }
+  }
```

Comment:
Notes to self: Should be handled via redux

Reply:
The real issue here is this: https://medium.com/@peter.yun.kim/controlled-and-uncontrolled-input-values-in-react-907119cc98d4

We sort of need it to be controlled in the case where the theme changes, which causes a re-render with a new `initialValue`

Reply:
It can be a controlled value whether we use redux or not. Since the value can be changed by multiple components, it makes sense to have redux control the changes.